package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -928,6 +931,8 @@ class SurvivalProbability(object):
        The `stop` keyword as passed to :meth:`SurvivalProbability.run` has now
        changed behaviour and will act in an `exclusive` manner (instead of it's
        previous `inclusive` behaviour),
+    .. versionchanged:: 2.7.0
+       Updated docs to align with discrete autocorrelation function.
```

Comment:
@hmacdope Do we need to document docs changes/improvements as part of `versionchanged`?

Reply:
Normally I would say no, but in this case I would say makes sense as the previous expression wan't correct, this way if someone asks us why it was different previously we can point to this.

Reply:
@RMeli If you are happy you can approve and then I will merge.

Reply:
I am going to speak for @RMeli as he is busy atm.